src/context/ListContext.jsx

```diff
@@ -48,5 +48,9 @@ Provider.defaultProps = {
   listB: [],
 };
 
-export const Context = createContext({});
-
+const Context = createContext({});
+const useListContext = () => useContext(Context);
+export {
+  Provider as ListContextProvider,
```

Comment:
Why don't you name the provider ```ListContextProvider``` in the first place when you create it? you don't need the ```as``` keyword...